spec/features/user_can_edit_their_items_and_reviews_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+feature "user can delete their reviews and venues" do
```

Comment:
Looks like a copy-paste issue, here. The name of this feature doesn't jive with the filename.